docs/faq.md

```diff
@@ -18,9 +27,29 @@ assertions. The ability to search for elements (using search actions) makes test
 UI changes. For example, EarlGrey provides APIs that allow searching for elements in scrollable
 containers, regardless of the amount of scrolling required.
 
+**Why do the tests have the application scaled with borders around it? How can I get them to fit in
+the video frame?**
+
+For your tests to have the application properly scaled, make sure the app under test has correct
+launch screen images present for all supported devices (see 
+[iOS Developer Library, Launch Files](https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/MobileHIG/LaunchImages.html)).
+
+**Why does EarlGrey need to modify the test's scheme and add a Copy Files Build Phase?**
```

Comment:
Is this really one of the FAQ?

In general, we shouldn't add Q's to FAQ section unless they are actually asked frequently. For one-time questions SO is the better option.